content/help/ops/security/repairing-citadel/index.md

```diff
@@ -8,16 +8,17 @@ keywords: [security,citadel,ops]
 If you suspect Citadel isn't working properly, verify the status of the `istio-citadel` pod:
 
 {{< text bash >}}
-$ kubectl get pod -n istio-system
+$ kubectl get pod -l istio=citadel -n istio-system
 NAME                                     READY     STATUS   RESTARTS   AGE
-istio-citadel-ff5696f6f-ht4gq              1/1    Running          0   25d
+istio-citadel-ff5696f6f-ht4gq            1/1       Running  0          25d
 {{< /text >}}
 
-If the `istio-citadel` pod does not exist, you should try to re-run `istio-demo.yaml`.
+If `istio-citadel` pod does not exist, you should try to re-deploy it.
 
-If `istio-citadel` is present, but its status is not `Running`, try the commands below to get more debugging information:
+If `istio-citadel` is present, but its status is not `Running`, try the commands below to get more
+debugging information and check if there are any errors:
```

Comment:
If the `istio-citadel` pod is present but its status is not `Running`, run the commands below to get more
debugging information and check if there are any errors:
I added the missing article and noun. To avoid an "expectation to fail", I replaced "try" with "run".
Consider splitting the commands into two different numbered steps.